packages/java/parser-jvm-plugin-pageable/src/main/java/dev/hilla/parser/plugins/pageable/PageablePlugin.java

```diff
@@ -0,0 +1,47 @@
+package dev.hilla.parser.plugins.pageable;
+
+import java.util.stream.Stream;
+
+import javax.annotation.Nonnull;
+
+import dev.hilla.parser.core.Plugin;
+import dev.hilla.parser.core.PluginsToolset;
+import dev.hilla.parser.core.RelativeClassInfo;
+import dev.hilla.parser.core.SharedStorage;
+import dev.hilla.parser.plugins.backbone.BackbonePlugin;
+import dev.hilla.parser.utils.PluginException;
+
+public class PageablePlugin implements Plugin.Transformer {
```

Comment:
`Plugin.Transformer` seems not available yet. Intentional?

If not, how about introducing the entire pageable module as a separate PR?